test/types/records/init/copy/copyWithDefaultArgs.chpl

```diff
@@ -0,0 +1,31 @@
+record R {
+  var x: int;
+
+  proc init(x: int, y: bool = true) {
+    if y then
+      this.x = x;
+  }
+
+  proc init(r: R, y: bool = true) {
```

Comment:
It seems odd to me that there are two duplicate copies of `init` here - wouldn't that lead to ambiguity if everything was "working" ?

Reply:
Not for these two, you can distinguish them based on the argument without a default value (one only takes an int for the first argument, the other only takes an instance of R for the first argument)

Reply:
Ah, I missed that for some reason. Thanks.